packages/synapse-react-client/src/components/EntityLink.tsx

```diff
@@ -15,7 +15,7 @@ type EntityLinkProps = {
   entity: string | EntityHeader | Entity
   versionNumber?: number
   /** Whether the component should link to the entity page in Synapse. Default true */
-  link?: boolean
+  link?: boolean | string
```

Comment:
Please update the comment to indicate that the link itself can be overridden by passing a string